AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -0,0 +1,72 @@
+// Created by Lunar on 30/09/2021.
+//
+
+import Foundation
+import Charts
+
+final class ChartViewModel: ObservableObject {
+    @Published var entries: [ChartDataEntry] = []
+    
+    private let stream: MeasurementStreamEntity
+
+    private var timeUnit: Double {
+        stream.session.type == .mobile ? 60 : 60*60
+    }
+    
+    private var mainTimer: Timer?
+    private var firstTimer: Timer?
+    
+    deinit {
+        mainTimer?.invalidate()
+        firstTimer?.invalidate()
+    }
+    
+    init(stream: MeasurementStreamEntity) {
+        self.stream = stream
+        generateEntries()
+        startTimers(stream.session)
+    }
+    
+    private func startTimers(_ session: SessionEntity) {
+        let sessionStartTime = session.startTime!
+        let secondsUntilFullMinute = timeUnit - Double(Int(Date().timeIntervalSince(sessionStartTime)) % Int(timeUnit))
```

Comment:
this name only makes sense for mobile sessions.
Also, it's hard to understand from the code what is happening here (why are we calculating this). Maybe some comment?..